src/CSSList/CSSBlockList.php

```diff
@@ -23,13 +23,23 @@ abstract class CSSBlockList extends CSSList
     /**
      * Gets all `DeclarationBlock` objects recursively, no matter how deeply nested the selectors are.
      *
-     * @return array<int, DeclarationBlock>
+     * @return list<DeclarationBlock>
      */
     public function getAllDeclarationBlocks(): array
     {
-        /** @var array<int, DeclarationBlock> $result */
+        /** @var list<DeclarationBlock> $result */
```

Comment:
Is this still needed?

Reply:
I've removed it for now. If PHPStan complains (on a higher check level), we can re-add it.

Reply:
It may have been needed when the array was passed by reference as an argument to `allDeclarationBlocks()`.